### PR TITLE
Enable Stackdriver Debugger by default

### PR DESCRIPTION
**Pull request by chanseokoh**
Do the same thing in #219 for the master branch.

### CODE REVIEW

File: gke-debian-openjdk/README.md [hunk 1]
```diff
@@ -38,13 +38,13 @@ The entry point for the gke-debian-openjdk image is [docker-entrypoint.bash](htt
...
-|----------------|--------------|---------------|----------------|---------------|
-| ALPN           | /opt/alpn/   | $ALPN_ENABLE  | $ALPN_DISABLE  | $ALPN_BOOT    |
-| Cloud Debugger | /opt/cdbg/   | $DBG_ENABLE   | $CDBG_DISABLE  | $DBG_AGENT    |
-| Cloud Profile  | /opt/cprof/  | $CPROF_ENABLE | $CPROF_DISABLE | $PROF_AGENT   |
-| Temporary file |              | $TMPDIR       |                | $SET_TMP      |
-| Java options   |              | $JAVA_OPTS    |                | $JAVA_OPTS    |
+| Feature        | directory    | Enable            | Disable        | JVM args      |
+|----------------|--------------|-------------------|----------------|---------------|
+| ALPN           | /opt/alpn/   | $ALPN_ENABLE      | $ALPN_DISABLE  | $ALPN_BOOT    |
+| Cloud Debugger | /opt/cdbg/   | \<on by default\> | $CDBG_DISABLE  | $DBG_AGENT    |
```
Comment (ludoch) [inline gke-debian-openjdk/README.md]: Is is a good idea to enable a debugger by default? 
I do not think so. Do we have security team review/approval?

**Comment by ludoch** [comment]
Does the debugger work on AWS as well or it is GCE centric?

**Comment by ludoch** [comment]
Images pushed with tag 0.97

Can you  tried? Just tried a compat runtime with this dockerfile, but could not see the debugger. Maybe it is my setting (no source code yet).

FROM gcr.io/google_appengine/jetty9-compat:0.97

ADD . /app

Once validated, I can make the 0.97 the latest on the 3 images.

**Comment by chanseokoh** [comment]
The changes are in, but I noticed it doesn't work in production. I will follow up this in #213.

**Pull request merged**